hyperapp.d.ts

```diff
@@ -26,6 +26,11 @@ export interface Component<Props> {
   (props: Props, children: VNodeChild<{} | null>[]): VNode<{}>
 }
 
+export type VNodeChildren =
+  | Array<VNodeChild<{} | null> | number>
+  | VNodeChild<{} | null>
+  | number
```

Comment:
Why is this not included with `VNodeChild`?

Reply:
This type is just an input type to the `h()` function, whereas `VNodeChild` is the type on the node itself (the `h()` functions accepts arbitrarily nested array of `VNode | string | number` and outputs a `VNode` with `VNode.children: VNodeChild[] (= Array<VNode | number>)`